src/v2/providers/storage.ts

```diff
@@ -174,6 +174,13 @@ export interface CustomerEncryption {
   keySha256?: string;
 }
 
+interface WithBucket {
+  /** The name of the bucket containing this object. */
+  bucket: string;
+}
+
+export type StorageEvent = CloudEvent<StorageObjectData, WithBucket>;
```

Comment:
Oh interesting, I didn't realize that storage also uses custom extension attributes.